public/js/views/management/subscriptions.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, PropTypes } from 'react';
+import SubscriptionList from 'components/subscription-list';
+
+import { gettext } from 'utils';
+
+
+export default class Subscriptions extends Component {
+
+  static propTypes = {
+    getUserSubscriptions: PropTypes.func.isRequired,
+    userSubscriptions: PropTypes.array.isRequired,
+  };
+
+  componentDidMount() {
+    if (this.props.userSubscriptions === null) {
+      this.props.getUserSubscriptions();
+    }
```

Comment:
Hopefully if some action changes a subscription, as long as it updates the data, the data will be correct.